content/blog/2024/12/16/2024-12-16-google-summer-of-code-2025-volunteers-needed-to-mentor-future-jenkins-contributors.adoc

```diff
@@ -0,0 +1,76 @@
+These GSoC contributors will get to learn from experienced open-source developers while writing code for real-world projects!
+A small stipend is provided as an incentive to support their open-source contributions.
+Refer to the link:https://summerofcode.withgoogle.com/get-started[GSoC contributor eligibility] documentation for more details.
+
+== Why might you consider being a Jenkins in GSoC mentor? 
+
+Mentor interaction is a vital part of GSoC.
+Mentoring is a great opportunity to improve your management and people (or general “soft”) skills while giving back to the community.
+In return for mentoring, a GSoC contributor works on your project full-time for 10-22 weeks.
+Think about the projects that you've always wanted to do but never had the time to complete.
+GSoC is a fantastic program and the Jenkins project is looking forward to participating in it again in 2025, potentially with your participation!
+
+== What does mentoring involve?
+
+Potential mentors are invited to read the link:https://www.jenkins.io/projects/gsoc/mentors[information for mentors].
+Note that being a GSoC mentor does not require expert knowledge of Jenkins.
+Mentors do not work alone.
+We make sure that every project has at least two mentors.
+GSoC org admins will be there to help to find technical advisers, so you can study together with your GSoC contributor.
+Mentoring takes about 5 to 8 hours of work per week (more at the start, less at the end).
+Mentors provide guidance, coaching, and sometimes a bit of cheerleading.
+They review GSoC contributor proposals, pull-requests, and contributor presentations at the evaluation phase.
+They fill in the Google-provided final evaluations at the end of the coding period.
+
+== So you want to be a mentor but don’t have a project idea? We can help with that!
+
+GSoC project ideas are coding projects that potential GSoC contributors are expected to accomplish in about 10-22 weeks.
+The coding projects can be new features, plugins, test frameworks, infrastructure, graphical interface, etc.
+If you don't have a specific project idea, please consider being a mentor for one of link:https://www.jenkins.io/projects/gsoc/2025/project-ideas/[these project ideas]. 
+If you have a specific project in mind, please send us your project ideas before the beginning of February, so they can get a proper review by the GSoC committee and by the community.
+Of course, we would love it if you were the mentor for your project idea.
+
+== How to submit a project idea
+
+Create a pull request with your idea in a .adoc file in the link:https://github.com/jenkins-infra/jenkins.io/tree/master/content/projects/gsoc/2025/project-ideas[project ideas].
+It is not necessary to submit a Google Doc, but it will still work if you want to do that.
+Refer to the link:https://www.jenkins.io/projects/gsoc/proposing-project-ideas/[instructions] on submitting ideas which include an .adoc template and some examples.
```

Comment:
```suggestion
Create a pull request with your idea in an .adoc file in the link:https://github.com/jenkins-infra/jenkins.io/tree/master/content/projects/gsoc/2025/project-ideas[project ideas].
It is not necessary to submit a Google Doc, but it will still work if you want to do that.
Refer to the link:https://www.jenkins.io/projects/gsoc/proposing-project-ideas/[instructions] on submitting ideas, which include an .adoc template and some examples.
```
Small grammar and punctuation adjustments